.travis.yml

```diff
@@ -25,10 +31,12 @@ env:
 
 before_install:
   - composer self-update
+  - if [ "$SYMFONY_VERSION" != "" ]; then composer require --no-update symfony/symfony:$(SYMFONY_VERSION); fi;
 
 install:
   - if [ "$deps" = "no" ]; then composer install; fi;
   - if [ "$deps" = "low" ]; then composer --prefer-lowest --prefer-stable update; fi;
+  - if [ "$deps" = "update" ]; then composer --prefer-dist update; fi;
```

Comment:
no need for ``deps=update``: we don't have a lock file in the repo, so install and update are doing the same already